lighthouse-core/gather/drivers/driver.js

```diff
@@ -210,6 +211,42 @@ class Driver {
   }
 
   /**
+   * Checks all serviceworkes and see if any duplications are running
+   * @param {string} pageUrl
+   */
+  checkForMultipleServiceWorkers(pageUrl) {
+    // Get necessary information of serviceWorkers
+    const getRegistrations = new Promise(resolve => {
+      this.once('ServiceWorker.workerRegistrationUpdated', resolve);
+    });
+    const getVersions = this.getServiceWorkerVersions();
+
+    return Promise.all([getRegistrations, getVersions]).then(res => {
+      const registrations = res[0].registrations;
+      const versions = res[1].versions;
+
+      let activeServiceWorkers = [];
+      registrations
+        .filter(reg => !reg.isDeleted)
+        .filter(reg => parseURL(reg.scopeURL).hostname === parseURL(pageUrl).hostname)
+        .forEach(reg => {
+          activeServiceWorkers = activeServiceWorkers.concat(versions
+             .filter(ver => ver.registrationId === reg.registrationId)
+          );
+        });
+
+      if (activeServiceWorkers.length > 1) {
+        const activeSW = activeServiceWorkers[0].scriptURL;
+        throw new Error(
+           `Unable to kill ServiceWorker(${activeSW}).`
```

Comment:
need a new error message since you aren't killing any service workers here :) Is there anything wrong with generating the `'You probably have multiple tabs open.'` message here?